invenio_rdm_records/templates/semantic-ui/invenio_notifications/grant-user-access.create.jinja

```diff
@@ -11,18 +11,36 @@
 %}
 
 {%- block subject -%}
-{{ _("👤✔️ You were granted {permission} access to record '{record_title}'").format(permission=permission, record_title=record_title) }}
+    {%- if record.is_published -%}
+        {{ _("👤✔️ You were granted {permission} access to record '{record_title}'").format(permission=permission, record_title=record_title) }}
```

Comment:
@anikachurilova @ntarocco @jbenito3 @kpsherva @jrcastro2 should we also mention here that you have `{permission}` to all versions of record/draft?